Update annotations docs (#170)
MicrosoftDocs/OData-docs: Odata-docs/client/annotation-support.md

# Client Annotation Support
**Applies To**: [!INCLUDE[appliesto-odataclient](../includes/appliesto-odataclient-v7.md)]

> Before ODataLib 6.10.0, OData core library supported metadata annotations for elements in the model and instance annotations for particular instances in the payload. But on client side, there wasn’t a good way to access these annotations. ODataLib 6.10.0 introduced several APIs to enable user to access annotations on client side. Basically, OData client follows the rules defined in [OData V4.0 protocol](https://docs.oasis-open.org/odata/odata/v4.0/errata02/os/complete/part1-protocol/odata-v4.0-errata02-os-part1-protocol-complete.html) (see [6.4 Vocabulary Extensibility](https://docs.oasis-open.org/odata/odata/v4.0/errata02/os/complete/part1-protocol/odata-v4.0-errata02-os-part1-protocol-complete.html#_Toc406398214)) to access instance or metadata annotations.

Reviewer: ```suggestion
Before ODataLib 6.10.0, OData core library supported metadata annotations for elements in the model and instance annotations for particular instances in the payload. But on client side, there wasn’t a good way to access these annotations. ODataLib 6.10.0 introduced several APIs to enable users to access annotations on client side. Basically, OData client follows the rules defined in [OData V4.0 protocol](https://docs.oasis-open.org/odata/odata/v4.0/errata02/os/complete/part1-protocol/odata-v4.0-errata02-os-part1-protocol-complete.html) (see [6.4 Vocabulary Extensibility](https://docs.oasis-open.org/odata/odata/v4.0/errata02/os/complete/part1-protocol/odata-v4.0-errata02-os-part1-protocol-complete.html#_Toc406398214)) to access instance or metadata annotations.
```

Author: #Resolved

> Before ODataLib 6.10.0, OData core lib has supported metadata annotations for metadata element in model and instance annotations for a particular instance in payload. But on client side, there isn’t a good way to get these  annotations. Since ODataLib 6.10.0, we provided several APIs to enable user to get annotations on client side. Basically, OData client follows the rules defined in [OData V4.0 protocol](https://docs.oasis-open.org/odata/odata/v4.0/errata02/os/complete/part1-protocol/odata-v4.0-errata02-os-part1-protocol-complete.html) (see [6.4 Vocabulary Extensibility](https://docs.oasis-open.org/odata/odata/v4.0/errata02/os/complete/part1-protocol/odata-v4.0-errata02-os-part1-protocol-complete.html#_Toc406398214)) to get instance annotations or metadata annotations.

Reviewer 2: ```suggestion
Before ODataLib 6.10.0, OData core lib has supported metadata annotations for metadata element in model and instance annotations for a particular instance in payload. But on client side, there wasn’t a good way to get these annotations. Since ODataLib 6.10.0, we provided several APIs to enable user to get annotations on client side. Basically, OData client follows the rules defined in [OData V4.0 protocol](https://docs.oasis-open.org/odata/odata/v4.0/errata02/os/complete/part1-protocol/odata-v4.0-errata02-os-part1-protocol-complete.html) (see [6.4 Vocabulary Extensibility](https://docs.oasis-open.org/odata/odata/v4.0/errata02/os/complete/part1-protocol/odata-v4.0-errata02-os-part1-protocol-complete.html#_Toc406398214)) to get instance annotations or metadata annotations.
```

Reviewer: If were removing the article `a`, shouldn't `instance` be pluralized?

Author: #Resolved

## How to get annotations on client side 

> All client CLR types in this tutorial are generated by OData Connected Service. Before we dive into this tutorial, checkout [OData Connected Service](/odata/connectedservice/getting-started) tutorial on how to generate the client side proxy classes.

Reviewer 2: ```suggestion
All client CLR types in this tutorial are generated using OData Connected Service extension. Before we dive into this tutorial, checkout [OData Connected Service](/odata/connectedservice/getting-started) tutorial on how to generate the client side proxy classes.
```

Reviewer: Perhaps add article `the` to make it `using the OData Connected Service extension`

Author: #Resolved

> All client CLR types in this tutorial are generated using the OData Connected Service extension. Before we dive into this tutorial, checkout [OData Connected Service](/odata/connectedservice/getting-started) tutorial on how to generate the client side proxy classes.

Reviewer: ```suggestion
All client CLR types in this tutorial are generated using the OData Connected Service extension. Before we dive into this tutorial, checkout the [OData Connected Service](/odata/connectedservice/getting-started) tutorial on how to generate the client side proxy classes.
```

Author: #Resolved

> All client CLR types in this tutorial are generated using the OData Connected Service extension. Before we dive into this tutorial, checkout [OData Connected Service](/odata/connectedservice/getting-started) tutorial on how to generate the client side proxy classes.

Reviewer 2: ```suggestion
All client CLR types in this tutorial are generated using the OData Connected Service extension. Before we dive into this tutorial, check out [OData Connected Service](/odata/connectedservice/getting-started) tutorial on how to generate the client side proxy classes.
```

Author: #Resolved

> OData Client provides the following APIs for getting annotations in `DataServiceContext` class.

Reviewer 2: ```suggestion
OData Client provides the following APIs for accessing annotations in `DataServiceContext` class.
```

Author: #Resolved
